docs/lib/geo/fragments/js/maps.md

```diff
@@ -27,13 +27,15 @@ Amplify.configure(awsconfig);
 ```
 
 Next, create and render the [Map](https://maplibre.org/maplibre-gl-js-docs/api/map/) with the help of [AmplifyMapLibreRequest's](https://github.com/aws-amplify/maplibre-gl-js-amplify/blob/main/API.md#amplifymaplibrerequest).
+
+Use the same `region` that you chose in the `amplify-cli` setup. This can also be found in your `aws-exports.js` file
 ```javascript
```

Comment:
```suggestion
Use the same `region` that you chose in the `amplify-cli` setup. This can also be found in your `aws-exports.js` file

```javascript
```